docs/06-contributors/999-rfcs/000-stories/story-14.md

```diff
@@ -71,7 +71,7 @@ resource TaskList {
    */
   inflight add_estimation(id: str, effort_estimation: duration): str {
     let j = Json.clone_mut(this.get_task(id));
-    j.set("effort_estimation", effort_estimation);
+    j.set("effort_estimation_in_seconds", effort_estimation.seconds);
```

Comment:
We're currently unable to use duration literals in inflight: [Unable to use duration literals in inflights](https://github.com/winglang/wing/issues/1629)

Reply:
Did we decide not to put this in the sprint?

Reply:
I think so. @Chriscbr is currently checking if the issue above is an easy fix. If not, we'll remove it from this story

Reply:
It'll take us time to get to add duration. I think we should remove story 14 altogether as it doesn't add much at this point.